### PR TITLE
[DX-1577, TT-12685] Tyk Charts 1.6 Release

### PR DESCRIPTION
**Pull request by carolwkc**
### For internal users - Please add a Jira DX PR ticket to the subject!

[DX-1577](https://tyktech.atlassian.net/browse/DX-1577)

---

### Preview Link

<br>

[preview](https://deploy-preview-5136--tyk-docs.netlify.app/docs/nightly/tyk-stack/)

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have reviewed the suggestions made by our AI (PR Agent) and updated them accordingly (spelling errors, rephrasing, etc.)
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labeled** the PR

___

### **PR Type**
documentation

___

### **Description**
- Added comprehensive release notes for Tyk Charts version 1.6.
- Documented breaking changes, dependencies, deprecations, upgrade instructions, and release highlights.
- Included detailed changelog with new features, updates, and fixed issues.
- Provided download links and further information for users.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>version-1.6.md</strong><dd><code>Added release notes for Tyk Charts version 1.6</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/content/product-stack/tyk-charts/release-notes/version-1.6.md

<li>Added new release notes for Tyk Charts version 1.6.<br> <li> Included sections for breaking changes, dependencies, deprecations, <br>upgrade instructions, release highlights, and changelog.<br> <li> Detailed new features, updates, and fixed issues.<br> <li> Provided download links and further information.<br>

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5136/files#diff-4eeae2d28dbd504e495e67ef6d3252620e3a2738381a442734a0837caa5ea94e">+240/-0</a>&nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

[DX-1577]: https://tyktech.atlassian.net/browse/DX-1577?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: tyk-docs/content/product-stack/tyk-charts/release-notes/version-1.6.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+
+### Support Lifetime
+<!-- Required. replace X.Y with this release and set the correct quarter of the year -->
+Our minor releases are supported until our next minor comes out. 
+
+---
+
+## 1.6.0 Release Notes
+
+##### Release Date XX August 2024
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-charts/release-notes/version-1.6.md]: Remember to add release date once confirmed

File: tyk-docs/content/product-stack/tyk-charts/release-notes/version-1.6.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+helm repo update
+
+helm upgrade [RELEASE_NAME] tyk-helm/[CHART_NAME]
+```
+
+#### Release Highlights
+<!-- Required. Use similar ToV to previous release notes. For example for a patch release: -->
+
+##### Updated MDCB Health check probes
+MDCB v2.7.0 release introduces /liveness and /readiness probes which give more accurate and detail health check information. MDCB dpeloyment has been updated to use the new endpoints. See [MDCB Health Check]({{<ref "tyk-multi-data-centre/setup-controller-data-centre#health-check">}}) section for information about the new probes.
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-charts/release-notes/version-1.6.md]: surround endpoints by backticks ` `, e.g. `/liveness` and check done in remaining content of release notes

File: tyk-docs/content/product-stack/tyk-charts/release-notes/version-1.6.md [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+| ---------------------------------------------------------- | ---------------------- | ---------------------- | -------- | 
+| [Kubernetes](https://kubernetes.io)                        | 1.26.x, 1.27.x, 1.28.x, 1.29.x | 1.19+          |          | 
+| [Helm](https://helm.sh)                                    | 3.14.x                 | 3.x                    |          | 
+| [Redis](https://redis.io)                                  | 6.2.x, 7.x    | 6.2.x, 7.x    | Used by Tyk Gateway and Dashboard | 
+| [MongoDB](https://www.mongodb.com/try/download/community)  | 5.0.x, 6.0.x, 7.0.x | 5.0.x, 6.0.x, 7.0.x | Used by Tyk Dashboard, Pump, and MDCB | 
+| [PostgreSQL](https://www.postgresql.org/download/)         | 12.x - 16.x        | 12.x - 16.x            | Used by Tyk Dashboard, Pump, and MDCB | 
+
+Given the time difference between your upgrade and the release of this version, we recommend customers verify the ongoing support of third-party dependencies they install, as their status may have changed since the release.
+
+#### Deprecations
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-charts/release-notes/version-1.6.md:79]: Should the MDCB deprecations of health check endpoints be added here?

File: tyk-docs/content/product-stack/tyk-charts/release-notes/version-1.6.md [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+
+Each change log item should be expandable. The first line summarises the changelog entry. It should be then possible to expand this to reveal further details about the changelog item. This is achieved using HTML as shown in the example below. -->
+
+<ul>
+
+<li>
+<details>
+<summary>MDCB: Added option to configure healthcheck cache renewal period</summary>
+
+Added `mdcb.healthcheck.cache_renewal_period` which configures the time interval (in seconds) at which the healthchecker refreshes its cached health status information (redis and DB). Default to 10 (seconds).
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-charts/release-notes/version-1.6.md]: Capitalise Redis

File: tyk-docs/content/product-stack/tyk-charts/release-notes/version-1.6.md [hunk 5]
```diff
@@ -0,0 +1,250 @@
...
+
+Each change log item should be expandable. The first line summarises the changelog entry. It should be then possible to expand this to reveal further details about the changelog item. This is achieved using HTML as shown in the example below. -->
+
+<ul>
+
+<li>
+<details>
+<summary>MDCB: Updated liveness and readiness probes</summary>
+
+Updated MDCB liveness and readiness probes to /liveness and /readiness respectively. These endpoints are available from MDCB v2.7.0. If you are deploying an earlier version of MDCB, please update the paths to /health in values.yaml file.
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-charts/release-notes/version-1.6.md]: Add link to new healthcheck docs here, links have been added to  MDCB 2.7 release notes PR as an example

File: tyk-docs/content/product-stack/tyk-charts/release-notes/version-1.6.md [hunk 6]
```diff
@@ -0,0 +1,250 @@
...
+</ul>
+
+##### Fixed
+<ul>
+
+<li>
+<details>
+<summary>Portal: Ingress resource should not be created if not enabled</summary>
+
+Fixed the issue that when Developer Portal component is enabled, an Ingress resource is being created, although portal ingress is not enabled.
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-charts/release-notes/version-1.6.md]: Capitalise portal

**Review by dcs3spp — CHANGES_REQUESTED** [review]
Thanks for submitting release notes PR @caroltyk @komalsukhani 

PR LGTM. I have fixed the CI checks and reviewed

File: tyk-docs/content/product-stack/tyk-charts/release-notes/version-1.6.md (hunk 3 above)
Comment (komalsukhani) [reply tyk-docs/content/product-stack/tyk-charts/release-notes/version-1.6.md:79]: New endpoints are added in MDCB v2.7.0. Old endpoints are still valid for older version.

**Comment by komalsukhani** [comment]
@dcs3spp I have implemented the changes. Just release date is missing. I will confirm with team and update it.

**Review by dcs3spp — CHANGES_REQUESTED** [review]
Thanks @caroltyk PR LGTM subject to release date being added. @komalsukhani Could you peer review and if ok approve?

**Review by dcs3spp — APPROVED** [review]
Approved subject to release date being added

**Review by komalsukhani — APPROVED** [review]

**Pull request merged**